cmd/explaintest/r/subquery.result

```diff
@@ -6,7 +6,7 @@ set session tidb_hashagg_partial_concurrency = 1;
 set session tidb_hashagg_final_concurrency = 1;
 explain select * from t1 where t1.a in (select t1.b + t2.b from t2);
 id	estRows	task	access object	operator info
-HashJoin_8	8000.00	root		CARTESIAN semi join, other cond:eq(test.t1.a, plus(test.t1.b, test.t2.b))
```

Comment:
The `CARTESIAN` keyword should not be removed in this case since its two children are full scans.